web/src/queries/storage/config-model.ts

```diff
@@ -91,6 +91,21 @@ function allMountPaths(drive: configModel.Drive): string[] {
   return drive.partitions.map((p) => p.mountPath).filter((m) => m);
 }
 
+// FIXME this logic does not belong here, but it is the lesser evil taking into account the
+// current code organization
+function mustStay(model: configModel.Config): boolean {
```

Comment:
BIG NP: Setting aside that this is marked as a hack, for final code, I’d suggest using a method that expresses the opposite of "stay", such as isDeletable (https://www.collinsdictionary.com/dictionary/english/deletable), canBeDeleted, or something similar.

While mustStay is shorter and straightforward too, it doesn’t quite fit with other attributes like isBoot, isExplicitBoot, etc. It feels a bit less natural as an attribute name in this context, IMHO.

Reply:
On the other hand, `if (mustStay) return` is definitely much more readable than `if (!canBeDeleted) return` :stuck_out_tongue_winking_eye: